pkg/config/config.go

```diff
@@ -146,7 +146,8 @@ func Scan(opts ...Option) (c *Config, err error) {
 		b, err := os.ReadFile(f)
 		if err == nil {
 			// best effort. skip lint checks
-			yaml.Unmarshal(b, c) //nolint:errcheck
+			yaml.Unmarshal(b, c)               //nolint:errcheck
+			yaml.Unmarshal(b, &c.originalData) //nolint:errcheck
 			if exists, header := HasHeader(string(b), ""); c.IsValid() || exists {
 				c.location = f
 				yaml.Unmarshal(b, &c.originalData) //nolint:errcheck
```

Comment:
Is this needed now?

Reply:
it is if we want to keep supporting cloud-config inputs without header (`#cloud-config`). After install now we push a `partition_info.yaml` file inside the oem folder - if the user doesn't use headers it would completely hide all the other fields gathered up to that point

Reply:
We refactored in a pairing session. Comment no longer applies.